### PR TITLE
Small Bugfixes: FactorizedReadout mask regularization loss, response & stimulus rate info in maheswaranathan config

### PR DESCRIPTION
**Pull request by thomasZen**
## Sum(model_output.shape) vs model_output.numel()

- model_output.numel() = the product of the dimensions = the total number of elements = 32 × 50 × 100 = 160,000.
- sum(model_output.shape) = the sum of the dimension sizes = 32 + 50 + 100 = 182.

This only affects how we log the learning rate during validation.

## Other changes
- Normalization changes for the sparse autoencoder (code we didn't use in the paper)
- Update openretina version to the most recent one (1.1.1) in uv.lock
- Consistency change for some regularizers
- Fixed rate for stimulus and response in maheswaranathan configs (30Hz -> 100Hz according to the loaded data which states dt of 10ms)

### CODE REVIEW

File: openretina/modules/readout/factorized.py [hunk 1]
```diff
@@ -159,7 +159,7 @@ def forward(self, x: torch.Tensor, **kwargs: Any) -> torch.Tensor:
     def regularizer(self, reduction: Optional[Literal["sum", "mean"]] = None) -> torch.Tensor:
         mask_r = self.reg[0] * torch.mean(torch.sum(torch.abs(self.mask_weights), dim=0))
         wt_r = self.reg[1] * torch.mean(torch.sum(torch.abs(self.feature_weights), dim=0))
-        reshaped_masked_weights = self.mask_weights.reshape(-1, 1, 1, self.mask_size[0], self.mask_size[1])
+        reshaped_masked_weights = self.mask_weights.T.reshape(-1, 1, 1, self.mask_size[0], self.mask_size[1])
```
Comment (thomasZen) [inline openretina/modules/readout/factorized.py:162]: Factorized readout was only used in configs/model/core_klindt_readout, not in the default model config.

File: uv.lock [hunk 2]
```diff
@@ -3374,7 +3374,7 @@ wheels = [
 
 [[package]]
 name = "openretina"
-version = "1.0.0"
+version = "1.1.1"
```
Comment (thomasZen) [inline uv.lock:3377]: we are at version 1.1.1 now already.

**Review by SamuelSuhai — APPROVED** [review]
Looks good.

File: openretina/modules/readout/factorized.py (hunk 1 above)
Comment (thomasZen) [reply openretina/modules/readout/factorized.py:162]: self.masks has the shape (num_pixels, num_neurons). Above command garbled the neurons and the pixels in a bad way, this is fixed by transposing the 2D matrix.
